### DISCUSSION TITLE
WIll there ever be a v86_64?

### DISCUSSION THREAD

**Discussion by not-amplify** [copy/v86#1122] [Ideas]
I think it intresting about how we can now emulate i386 in our web browsers with wasm. However i was wondering if it was ever possible (depending if the Browser Binary is 64 bit, along with the CPU) i was wondering if there could ever be a 64 bit v86?

**Comment by SuperMaxusa** [comment]
See https://github.com/copy/v86/issues/133 and https://github.com/copy/v86/issues/648

**Reply by not-amplify** [reply]
> See #133 and #648

thank you.
